### DISCUSSION TITLE
White screen and no reaction on Laptop after it was in standby

### DISCUSSION THREAD

**Discussion by JengaBenga** [SafeExamBrowser/seb-win-refactoring#493] [Q&A]
I set my Laptop in standby mode during a "practice" exam and  after i used the Laptop again the Safe exam Browser asked a Code. I didnt fill in the code and it automatically showed a white screen and nothing works anymore... Even after multiple restartes i just get the white screen and I cant do anything...

I believed i could close the Browser or type in the code, but I cant. 

I use windows 10 and the newest Version of the Safe exam browser.

**Comment by dbuechel** [comment]
The lock screen can be closed by entering the correct quit / unlock password, or simply by closing the lock screen in case there is no such password set for the current configuration.

If there should be any issue that your computer isn't responsive or alike, try hard resetting your machine (i.e. press the power button for at least 5 seconds to completely turn off your computer).
